stories/documentation/overlays/tooltip/tooltip.stories.html

```diff
@@ -1,3 +1,11 @@
+<style>
+	.short-button {
+		width: 150px;
+		overflow: hidden;
```

Comment:
`overflow`, `white-space` & `text-oveflow` could be replaced with a `u-ellipsis` class on the `<button>`